Code/GraphMol/new_canon.cpp

```diff
@@ -331,29 +403,57 @@ void getNbrs(const ROMol &mol, const Atom *at, int *ids) {
 }
 
 bondholder makeBondHolder(const Bond *bond, unsigned int otherIdx,
```

Comment:
Is there a reason this can't be a constructor?  This way, emplace_back could be used.

Reply:
no particular reason... it's just old code and could definitely be cleaned up